scripts/gif_maker_parallelized.py

```diff
@@ -0,0 +1,125 @@
+"""
+Script to generate gifs from traj
+
+Requirements:
+
+sudo apt update
+sudo apt install povray
+sudo apt install ffmpeg
+pip install ase==3.20
```

Comment:
Let's update this to 3.21.1 like we discussed on chat

Reply:
Any reason we don't just use 3.22.0?

Reply:
[But that's `3.21.*`, right?](https://github.com/Open-Catalyst-Project/ocp/blob/master/env.common.yml#L7)

Reply:
Ah sorry, it's just local then. Sounds good.